frontend/src/components/EntityList.vue

```diff
@@ -155,17 +155,17 @@ export default {
       this.selected = this.$refs.selector.getSelected();
     },
     onDeletion() {
-      const promises = this.selected.map(eId => {
-        this.$api.deleteEntity({
+      const promises = this.selected.map(async eId => {
+        await this.$api.deleteEntity({
```

Comment:
So, the original idea was to trigger all the async. API calls to delete entities and store their promises in the variable.

Later (in line 164) they are all awaited and only afterwards the API function to get entities is called.

To me it looks like the `.map()` call does not return promises, which makes the call to `Promise.all()` behave in an unintended way.

Maybe this would do the trick:

```suggestion
      const promises = this.selected.map(eId => {
        return this.$api.deleteEntity({
```

Reply:
Yes, you're right! Thanks!